views/BoggleView.java

```diff
@@ -450,17 +450,17 @@ private void launchControls(){
 //                    (this.model.getAllWords().length - this.model.getGameStats().getScore()));
 //            box1.getChildren().add(wordCount);
 //            box1.setAlignment(Pos.TOP_CENTER);
-//            String wordFound = "Words Found: ";
-//            for (String word: this.model.getGameStats().getPlayerWords()){
-//                wordFound +=  "\n" + word ;
-//            }
-//            Label wordsFound = new Label(wordFound);
-//            box3.getChildren().add(wordsFound);
-//            box3.setAlignment(Pos.BOTTOM_CENTER);
-//            VBox finalBox = new VBox(20, box1, box2, box3);
-//            borderPane.setRight(finalBox);
-//            finalBox.setAlignment(Pos.CENTER);
-//            finalBox.setStyle("-fx-background-color: rgb(225,225,255,0.52);");
+            String wordFound = "Words Found: ";
+            for (String word: this.model.getGameStats().getPlayerWords()){
+                wordFound +=  "\n" + word ;
+            }
+            Label wordsFound = new Label(wordFound);
+            box3.getChildren().add(wordsFound);
+            box3.setAlignment(Pos.BOTTOM_CENTER);
+            VBox finalBox = new VBox(20, box1, box2, box3);
+            borderPane.setRight(finalBox);
+            finalBox.setAlignment(Pos.CENTER);
+            finalBox.setStyle("-fx-background-color: rgb(225,225,255,0.52);");
```

Comment:
set finalBox.setMaxSize(400,400)
I am thinking this might help. If you have not done it yet.
Also 400 could be a large number.